slave.mk

```diff
@@ -230,6 +230,8 @@ $(info "BLDENV"                          : "$(BLDENV)")
 $(info "VS_PREPARE_MEM"                  : "$(VS_PREPARE_MEM)")
 $(info "ENABLE_SFLOW"                    : "$(ENABLE_SFLOW)")
 $(info "ENABLE_NAT"                      : "$(ENABLE_NAT)")
+$(info "SONIC_TELEMETRY_WRITABLE"       : "$(SONIC_TELEMETRY_WRITABLE)")
```

Comment:
Keeping in line with the other options, I suggest removing the `SONIC_` prefix from the user-facing option name.